test/tests/concrete_moisture_heat_transfer/tests

```diff
@@ -9,47 +9,596 @@
     rel_err = 1e-6
     abs_zero = 1e-10
   []
-  [thermal_properties]
-    issues = '#9'
-    requirement = 'Blackbear shall compute material properties for thermal transport in concrete '
-    [kodur_2004]
-      detail = 'using the Kodur 2004 model, '
+  issues = '#245'
+  design = 'ConcreteThermalMoisture.md'
+  [thermal_model]
+    requirement = 'Blackbear shall provide the moisture capacity and moisture diffusivity in domain as demonstrated by'
+    [constant_thermal]
+      type = 'CSVDiff'
+      input = 'concrete_thermal_moisture_properties.i'
+      csvdiff = 'constant_thermal_model_out.csv'
+      cli_args = 'Materials/concrete/thermal_model=CONSTANT '
+                  'Materials/concrete/ref_density=2231 '
+                  'Materials/concrete/ref_specific_heat=1100 '
+                  'Materials/concrete/ref_thermal_conductivity=3.0 '
+                  'Materials/concrete/moisture_model=Bazant '
+                  'Materials/concrete/D1=3.0e-10 '
+                  'Materials/concrete/n=16 '
+                  'Materials/concrete/critical_relative_humidity=0.75 '
+                  'Materials/concrete/coupled_moisture_diffusivity_factor=1e-5 '
+                 'Outputs/file_base=constant_thermal_model_out '
+      detail = 'Constant thermal model.'
+    []
+    [constant_thermal_ref_density]
```

Comment:
I think this would read better if you moved all the error tests into a separate requirement. You can keep them in the same file here, but add another block at the top level just as a container for the error tests.